yarn.lock

```diff
@@ -1732,7 +1732,7 @@
     i18next "^10.3"
     moment "^2.24.0"
 
-"@dhis2/prop-types@^1.6.4":
+"@dhis2/prop-types@1.6.4", "@dhis2/prop-types@^1.6.4":
```

Comment:
Side note: where is this "pinned" @dhis2/prop-types 1.6.4 coming from?

Reply:
Ah, I probably need to regenerate the lockfile. I'll check.

Reply:
Fixed, was a remnant of the install I did in ui-icons where I specified the exact version.